packages/tests/src/web-test-runner.config.ts

```diff
@@ -68,10 +68,14 @@ function makeSauceLauncherOnce() {
           ' and SAUCE_ACCESS_KEY environment variables.'
       );
     }
-    sauceLauncher = createSauceLabsLauncher({
-      user,
-      key,
-    });
+    sauceLauncher = createSauceLabsLauncher(
```

Comment:
Can we only update the `@web/test-runner-saucelabs` version in this PR?

Reply:
of course! only added it because previous fail, reverting now